### PR TITLE
[e2e] add a max rss for each connector

### PR DESCRIPTION
**Pull request by tarekziade**
- Adds a max rss of 200MiB (and more for some)

### CODE REVIEW

**Comment by tarekziade** [comment]
BK nightly build https://buildkite.com/elastic/connectors-python-nightly/builds/545

File: connectors/tests/ftest.sh [hunk 1]
```diff
@@ -47,9 +47,9 @@ if [[ $PERF8 == "yes" ]]
 then
     if [[ $PLATFORM == "darwin" ]]
     then
-      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil -c $ELASTIC_INGEST --debug & PID=$!
+      $PERF8_BIN --refresh-rate $REFRESH_RATE -t $ROOT_DIR/perf8-report-$NAME --asyncstats --memray --psutil --psutil-max-rss 209715200 -c $ELASTIC_INGEST --debug & PID=$!
```
Comment (timgrein) [inline connectors/tests/ftest.sh:50]: Could we add a comment, that 209715200 bytes are 200MiB?

**Comment by artem-shelkovnikov** [comment]
200MB is not enough, right? I see some connectors take almost 270 MB, so let's do 300?

File: connectors/tests/ftest.sh (hunk 1 above)
Comment (tarekziade) [reply connectors/tests/ftest.sh:50]: I think I'll do better : allow `--psutil-max-rss 200MiB`

**Comment by tarekziade** [comment]
> 200MB is not enough, right? I see some connectors take almost 270 MB, so let's do 300?

Yeah I did this on purpose to see some fail. They don't haha. Because the bash script does not get back the exit code. We need to tweak that script

I so want to get rid of that bash file and do a simpler Python script..

**Comment by tarekziade** [comment]
new run with raised limits per connector when needed https://buildkite.com/elastic/connectors-python-nightly/builds/554

**Review by navarone-feekery — APPROVED** [review]

**Pull request merged**